python/site-packages/sitecustomize.py

```diff
@@ -6,7 +6,8 @@
 # virtualenv-installed packages
 
 if "EBPYTHONPREFIXES" in os.environ:
-    postfix = os.path.join('lib', 'python'+sys.version[:3], 'site-packages')
+    pv = ".".join([str(sys.version_info.major), str(sys.version_info.minor)])
```

Comment:
we can use the same notation as upstream here (see python easyblock):
`postfix = os.path.join('lib', 'python' + '.'.join(map(str, sys.version_info[:2])), 'site-packages')`
same below.

Reply:
done in next commit